src/ui/partials/Animation.svelte

```diff
@@ -59,20 +61,20 @@
 -->
 <div id="tsparticles" class="animation" />
 
-{#if $animationType === "circuit"}
+{#if !$animationHidden && $animationType === "circuit"}
```

Comment:
Might as well name the variable/stored value `animationsShown` to avoid all those negations :D

Reply:
alright